src/OrleansTestingHost/TestingSiloHost.cs

```diff
@@ -661,13 +677,29 @@ public static SiloHandle StartOrleansSilo(TestingSiloHost host, Silo.SiloType ty
 
             // Load initial config settings, then apply some overrides below.
             ClusterConfiguration config = new ClusterConfiguration();
-            if (options.SiloConfigFile == null)
+            try
             {
-                config.StandardLoad();
+                if (options.SiloConfigFile == null)
+                {
+                    config.StandardLoad();
```

Comment:
Hi, I kept on thinking about it and although I'm still not fully convinced about the suggested breaking change, I do consider that it's a different change than this PR. So we should be able to merge this and have a separate PR with your suggested change after we do some local testing on how it would break existing users of the nuget package. That would also make evaluating #1643 easier, as it currently contains this commit too in it.